### ISSUE TITLE
listOpenTypeFeatures() causes traceback

### ISSUE THREAD

**Issue by justvanrossum** [typemytype/drawbot#103]
```python
font("ACaslonPro-Regular")
print listOpenTypeFeatures()
```

```
Traceback (most recent call last):
  File "<untitled>", line 2, in <module>
  File "/Users/just/code/git/drawbot/drawBot/drawBotDrawingTools.py", line 1514, in listOpenTypeFeatures
    fontName = self._dummyContext._state.text.fontName
AttributeError: 'FormattedString' object has no attribute 'fontName'
```

**Comment by typemytype** [comment]
fixed here c669953088d4e82621b84e2bcdf55ef77bb09b89

**Comment by justvanrossum** [comment]
Thanks!
